styles/globals.css

```diff
@@ -182,7 +200,16 @@ a {
 
 .remove {
   background-color: aliceblue;
+}
+
+.video-responsive {
```

Comment:
I remembered you mentioning that there were multiple of these classes, so we should def consider deleting duplicates.